llvm/test/MC/Disassembler/Xtensa/code_density.txt

```diff
@@ -0,0 +1,64 @@
+# RUN: llvm-mc -triple=xtensa -mattr=+density -disassemble < %s | FileCheck -check-prefixes=CHECK-DENSITY %s
+# RUN: llvm-mc -triple=xtensa -disassemble %s &> %t
+# RUN: FileCheck -check-prefixes=CHECK-CORE < %t %s
```

Comment:
--implicit-check-not=warning: is useful to ensure that there are no additional warnings